bears/jinja2/Jinja2Bear.py

```diff
@@ -312,16 +312,17 @@ def run(self,
             variable_spacing: int=1,
             control_spacing: int=1):
         """
-        Check Jinja2 templates for syntax, formatting and documentation issues.
+        Check `Jinja2 templates <http://jinja.pocoo.org>`_ for syntax,
+        formatting and documentation issues.
         The following aspects are being looked at:
 
         * Variable spacing:
             Variable tags should be padded with one space on each side, like
             this: ``{{ var_name }}``. This can be set to any number of spaces
-            via the setting variable_spacing. Malformatted variable tags are
-            detected and fixes suggested.
+            via the setting ``variable_spacing``.
+            Malformatted variable tags are detected and fixes suggested.
         * Control spacing:
-            Like variable spacing, but for control blocks, i.e. if and for
+            Like variable spacing, but for control blocks, i.e., if and for
```

Comment:
The comma after the `i.e.` [is apparently only needed in American English](https://jakubmarian.com/comma-after-i-e-and-e-g/). Just throwing this in there as a nice to know, if you don't already. ;)